tests/test_siamese_spectra_model.py

```diff
@@ -125,17 +125,17 @@ def test_model_training(simple_training_spectra):
                                     intensity_scaling=0.5, base_dims=(200, 200),
                                     embedding_dim=100,
                                     train_binning_layer=False,
-                                    same_prob_bins=np.array([(0, 1.0)]),
+                                    same_prob_bins=np.array([(0, 0.5), (0.5, 1.0)]),
```

Comment:
The last tests checking: assert (np.array(history["collection_targets"]) == 1).sum() == 200
did pass before. However, this was as far as I understand just coincidence, since no same_prob_bins were set. So the pair distribution can be random. Since we balance the inchikey picking better now, this was 266. 
I did correctly understand this test right?